jupyter-extension/jupyterlab_pachyderm/pfs_manager.py

```diff
@@ -51,7 +51,8 @@ class ContentModel(typing.TypedDict):
     Whether or not the entity can be written to. Currently, the Pachyderm
     extension is read-only for non-pipeline operations.
     """
-
+    file_uri: typing.Optional[str]
```

Comment:
Did you mean to commit this change? This model was intended to match the jupyter ContentModel API so I'm not sure we should be modifying this. Also I see you've updated one of the call sites for creating this object but there are several that haven't been updated.

Reply:
So I only included the `file_uri` for the case where we actually are paginating a directory. It's used in the `pagination_marker` parameter for `list_file` so it should be a valid Pachyderm File URI.

Good point about how `ContentModel` should mirror the jupyter ContentModel API though 🤔  It might be possible to construct these file URIs client-side with the information already provided for a file lemme think about that.

Reply:
That makes sense updated the other references to set it to None or str(fileinfo.file) where appropriate